site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -190,15 +190,27 @@ To upgrade FusionAuth when running with `docker-compose`:
 . Start it up: `docker-compose up`.
 . Login to the administrative UI. 
 
-If there were migrations required, they'll be applied automatically to your database if the installation link:/docs/v1/tech/installation-guide/fusionauth-app#runtime-modes[has a runtime mode of development]. If the installation is in production mode, make sure you apply the migrations out of band. 
+=== Migrations
+If there were migrations required, they'll be applied automatically to your database if the installation link:/docs/v1/tech/installation-guide/fusionauth-app#runtime-modes[has a runtime mode of development]. 
+
+==== Prior to 1.19
+
+If the installation is in `production` mode make sure you apply the migrations out of band. 
+
+==== Version 1.19 or later
+
+You can use `silent` mode for db upgrades in `development` or `production` runtime modes. The runtime modes are no longer used to identify if silent mode is desired, instead set the `fusionauth-app.silent-mode` configuration value to `true`. See the link:/docs/v1/tech/reference/configuration/[configation reference] or the link:/docs/v1/tech/guides/silent-mode/[silent mode guide] for more information.
+
+If you want to manage your own upgrades, performing the SQL migrations out of band is still a good option.
```

Comment:
Suggested text based upon my logic workflow comment (feel free to adjust)

(also fixed the link to end with a slash `../fusionauth-app/#runtime-modes`

```
When FusionAuth is configured to run in `development` ink:/docs/v1/tech/installation-guide/fusionauth-app#runtime-modes[runtime mode] you will enter maintenance mode on startup to complete any required database migrations. 

You may optionally perform this silently and not require any user interactive by enabling silent mode (link to config). 

When configured to run in `production` runtime mode you only enter maintenance mode if you have enabled silent mode . This ensures that when you are running in production an interactive maintenance mode screen is never shown to an end user. 

If you do not enable silent mode, you will need to perform database migrations out of band prior to starting FusionAuth after an upgrade. 


==== Version 1.19 or later

Prior to version 1.19.0, all database schema updates must be applied out of band when running in production runtime mode. 

When running in development runtime mode, silent mode was enabled based upon the presence of environment variables and could not explicitly be enabled or disabled. 

```